machine-setup.sh

```diff
@@ -16,9 +16,11 @@ else
     __ms_shell=sh
 fi
 
-target=""
+target="${MACHINE}"
 USERNAME=`echo $LOGNAME | awk '{ print tolower($0)'}`
 
+if [[ ${target} == "" ]] ; then
```

Comment:
This will skip all machines when building from rrfs-workflow.
Will this cause a problem or not?

Reply:
@EricJames-NOAA

Reply:
@guoqing-noaa we get target from rrfs-workflow's $MACHINE on line 19, so we don't need to run through all the following logic.  It is not reliable anyway, because we can't distinguish ursa from hera when we find we are on /scratch3 or /scratch4.

Reply:
So all those module initialization are not needed, right?

Reply:
Apparently not